### PR TITLE
add story_followers to specification file

### PR DESCRIPTION
**Pull request by simeon13**
Add Story Followers with create_attributes and associations to specification file.

### CODE REVIEW

File: spec/lib/mavenlink/story_follow_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "spec_helper"
+
+describe Mavenlink::StoryFollow, type: :model do
+  describe "associations" do
+    it { is_expected.to respond_to :story }
```
Comment (b6b) [inline spec/lib/mavenlink/story_follow_spec.rb:5]: We should also spec the follower association here to and add the shared model behavior:
```
it_should_behave_like "model", "story_follows"
```

File: spec/lib/mavenlink/story_follow_spec.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "spec_helper"
+
+describe Mavenlink::StoryFollow, type: :model do
```
Comment (b6b) [inline spec/lib/mavenlink/story_follow_spec.rb:3]: We should also add the shared behavior for models here
```
it_should_behave_like "model", "story_follows"
```

**Review by b6b — CHANGES_REQUESTED** [review]

**Review by kcotugno — APPROVED** [review]
LGTM! 👍

**Review by b6b — APPROVED** [review]
LGTM!

**Pull request merged**